benchmark/Streamly/Benchmark/Data/Parser/ParserD.hs

```diff
@@ -351,7 +357,8 @@ o_n_space_serial value =
     , benchIOSink value "sequenceA_/100" $ sequenceA_ (value `div` 100)
     , benchIOSink value "sequence/100" $ sequence (value `div` 100)
     , benchIOSink value "sequence_/100" $ sequence_ (value `div` 100)
-    , benchIOSink value "choice/100" $ choice (value `div` 100)
+    , benchIOSink value "choice (asum)/100" $ choiceAsum (value `div` 100)
+    , benchIOSink value "choice" $ choice (value `div` 100)
```

Comment:
Should name it to `choice/100` to indicate that this benchmark is 100 times shorter than usual.